rpm/gst-droid.spec

```diff
@@ -19,8 +18,7 @@ BuildRequires:  pkgconfig(gstreamer-tag-1.0)
 BuildRequires:  pkgconfig(nemo-gstreamer-interfaces-1.0) >= 0.20200421.0
 BuildRequires:  pkgconfig(libexif)
 BuildRequires:  meson
-BuildRequires:  droidmedia-devel >= 0.20200421.0
-BuildRequires:  pkgconfig(libandroid-properties)
+BuildRequires:  pkgconfig(droidmedia) >= 0.20200421.0
```

Comment:
The versioned dependency is not really needed any more because pkgconfig file is only provided by the new droidmedia version. Unless we want to update the version after merging droidmedia changes to tell the specific minimum version.

Reply:
Fixed, thank you.